sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/_azure_configuration_client_async.py

```diff
@@ -83,17 +83,31 @@ def __init__(
             base_user_agent=USER_AGENT, **kwargs
         )
 
-        pipeline = kwargs.get("pipeline")
         self._sync_token_policy = AsyncSyncTokenPolicy()
-
-        if pipeline is None:
-            aad_mode = not isinstance(credential, AppConfigConnectionStringCredential)
-            pipeline = self._create_appconfig_pipeline(
-                credential=credential, aad_mode=aad_mode, base_url=base_url, **kwargs
-            )
+        
+        aad_mode = not isinstance(credential, AppConfigConnectionStringCredential)
+        if aad_mode:
+            scope = base_url.strip("/") + "/.default"
+            if hasattr(credential, "get_token"):
+                credential_policy = AsyncBearerTokenCredentialPolicy(
+                    credential, scope
+                )
+            else:
+                raise TypeError(
+                    "Please provide an instance from azure-identity "
+                    "or a class that implement the 'get_token protocol"
+                )
+        else:
+            credential_policy = AppConfigRequestsCredentialsPolicy(credential)
 
         self._impl = AzureAppConfiguration(
-            credential, base_url, credential_scopes=self._credential_scopes, pipeline=pipeline  # type: ignore
+            credential, base_url,
+            credential_scopes=self._credential_scopes,
+            authentication_policy=credential_policy,
+            user_agent_policy=self._config.user_agent_policy,
+            per_call_policies=self._sync_token_policy,
+            transport=AioHttpTransport(),
```

Comment:
Do we really need to specify all these default settings?

Reply:
We need as we are not passing `pipline` now, these values will in use when setting self._config and self._client inside AzureAppConfiguration.
But your comment reminds me that there still have duplicate settings in this init, e.g. no longer need to create self._config in this _init.